testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeTransactionLogCache.java

```diff
@@ -63,7 +63,7 @@ public void testAllDataFilesAreLoadedWhenTransactionLogFileAfterTheCachedTableVe
         String tableName = "test_dl_cached_table_files_accuracy_" + randomTableSuffix();
         String tableDirectory = "databricks-compatibility-test-" + tableName;
 
-        onTrino().executeQuery(format("CREATE TABLE delta.default.%s (col INT) WITH (location = 's3://%s/%s')",
+        onTrino().executeQuery(format("CREATE TABLE delta.default.%s (col INT) WITH (location = 's3://%s/%s', checkpoint_interval = 10)",
```

Comment:
why?

Reply:
The default checkpoint interval is 100 in 11.3 LTS.